File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// Anapioficeandfire SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class AnapioficeandfireMakeContext
{
    public static function call(array $ctxmap, ?AnapioficeandfireContext $basectx): AnapioficeandfireContext
    {
        return new AnapioficeandfireContext($ctxmap, $basectx);
    }
}

File: rb/utility/make_context.rb
# Anapioficeandfire SDK utility: make_context
require_relative '../core/context'
module AnapioficeandfireUtilities
  MakeContext = ->(ctxmap, basectx) {
    AnapioficeandfireContext.new(ctxmap, basectx)
  }
end
